app/controllers/cards_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CardsController < ApplicationController
+  def index
+    @cards = Card.all
+  end
+
+  def create
+    card = Card.new(cards_params)
+    card['review_date'] = Date.today
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.